layouts/partials/head.html

```diff
@@ -118,6 +129,14 @@
 
     const init = (doc = document) => {
       // NOTE: everything within this callback will be executed for initial page navigation. This is a good place to put JavaScript that only replaces DOM nodes.
+      {{if $.Site.Data.config.enableCodeBlockCopy -}}
```

Comment:
because of L172 where we already do window.addEventListener for both of these, should be safe to not have the if case for `enableSPA` and have it be similar to the `enableLatex` one below

Reply:
You're right! Sorry, I should have adjusted this PR when I saw your changes. I lazily reopened the initial PR 😅

Reply:
Should be good now!